handbook/.pages

```diff
@@ -4,4 +4,5 @@ nav:
     - rbac
     - openid-connect
     - ldap
+    - data-stores
```

Comment:
The actual content in this section only mentions data stores one time -- the rest of the time we're talking databases, which I understand is a slightly narrower concept than data stores. For me, this raises 2 questions:

- Do we literally mean database every time we say database? Or do we sometimes mean data stores? 
- Do you plan to add non-Mongo stuff to this section for which "data store" would be more accurate? If so, I can understand keeping this section title as the slightly more general term.

I know they're often interchangeable, but I think it's worth being deliberate here.

Reply:
So, as I began putting this content together I realized it might make sense to have a group of topics on data stores (a general term covering databases, file storage, etc) where MongoDB is one sub topic. The other topics that would fit into this group would be file storage, MinIO, and Data Finder Index. This was a key piece I wanted to get your feedback on; e.g. the architecture of these collection of docs. If you build the handbook locally you can see how this would work. Currently its just the Mongo doc and a stub on file storage. 
- File storage -  I expect  this will be very light. Mostly just pointing back to SPD existing docs. Perhaps some discussion on encryption and a table with diagrams similar to what we see in this doc. 
- MinIO - this would be Timo's handbook chanter as it stands now. I made some comments about that in his PR. 
- DataFinder Index - Apparently we support putting this index on a dedicated database server. I want to document this because the performance of the index can be really important in SystemLink, but I need to set it all up first before we could write about it.